lib/irq.c

```diff
@@ -189,7 +189,17 @@ irqs_set_data_none(vfu_ctx_t *vfu_ctx, struct vfio_irq_set *irq_set)
     eventfd_t val;
 
     for (i = irq_set->start; i < (irq_set->start + irq_set->count); i++) {
-        efd = vfu_ctx->irqs->efds[i];
+        switch (irq_set->index) {
```

Comment:
Please convert this into a function and use it here and below.

Reply:
ok

Reply:
Not needed, just a separate commit which we'll squash when merging this PR.